### PR TITLE
Register 3d projection by default.

### PR DESCRIPTION
**Pull request by anntzer**
Now that we always install mpl_toolkits (#12317), we may as well always register
mplot3d and save an "apparently unused" import for the users (#12249).

## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

File: lib/matplotlib/projections/__init__.py [hunk 1]
```diff
@@ -70,3 +73,7 @@ def get_projection_names():
     Get a list of acceptable projection names.
     """
     return projection_registry.get_projection_names()
+
+
+# Register Axes3D.
```
Comment (tacaswell) [inline lib/matplotlib/projections/__init__.py]: Don't think we need this any more?

Comment (anntzer) [reply lib/matplotlib/projections/__init__.py]: oops, indeed, fixed

**Comment by anntzer** [comment]
they passed

**Comment by dstansby** [comment]
Worth a backport to `3.1.x`?

**Comment by ImportanceOfBeingErnest** [comment]
People are often concerned about the load/import time of matplotlib (#2188, #11546, ). Are the implications clear on this?

**Comment by anntzer** [comment]
Examining the output of `python -X importtime -c 'import matplotlib.pyplot, mpl_toolkits.mplot3d'` indicates that the slowdown should be on the order of 1% (~3ms vs 300ms).

**Pull request merged**